frontend/src/components/config/App.tsx

```diff
@@ -37,20 +38,22 @@ function App() {
   const dispatch = useAppDispatch();
   const [loading, setLoading] = useState(true);
 
+  const { firebaseUser } = useAppSelector((state) => state.account);
+
   // Track page view on every change in location and clear errors when switching pages
   useEffect(() => {
     ReactGA.pageview(location.pathname);
   }, [location]);
 
   // Set authentication status when Firebase auth status changes
   useEffect(() => {
-    app.auth().onAuthStateChanged((firebaseUser) => {
-      dispatch(setFirebaseUser(firebaseUser?.toJSON() as FirebaseUserType || null));
+    app.auth().onAuthStateChanged((user) => {
```

Comment:
What was your reasoning for changing this to `user`? I got confused between `firebaseUser` as the account object, and `user` as the `User` object, which is why I'm asking.

Reply:
The main reason was after importing `firebaseUser` from the Redux store, this name overshadowed that variable, causing an ESLint warning. Therefore I went with just renaming it to `user` instead

Reply:
Ah gotcha, sounds good 👍